lib/chef/project_cache.rb

```diff
@@ -98,13 +99,61 @@ def generate_combined_manifest
     def update_cache
       create_cache_dirs
       manifests_to_delete = local_manifests - project.manifests
-      manifests_to_fetch = project.manifests - local_manifests
       debug("Files to delete:\n#{manifests_to_delete.map{|f| "* #{f}"}.join("\n")}")
       debug("Files to fetch:\n#{manifests_to_fetch.map{|f| "* #{f}"}.join("\n")}")
       manifests_to_delete.each {|m| delete_manifest(m) }
       manifests_to_fetch.each {|f| fetch_manifest(f) }
     end
 
+    def manifests_to_fetch
+      @fetch_list ||= project.manifests.select { |manifest| manifest_fetchable?(manifest) }
+    end
+
+    def manifest_fetchable?(manifest)
```

Comment:
Yeah, you're right.  I was struggling with what to name it, and went through 3 different ones, because I :heart: bikesheds.  Changing...